apps/website/src/routes/docs/_components/status-banner/status-banner.tsx

```diff
@@ -17,7 +17,7 @@ function getMessageByStatus(status?: ComponentStatus) {
     case ComponentStatus.Ready:
       return (
         <>
-          This component is <strong>Production Readty</strong>
+          This component is <strong>Production Ready</strong>
```

Comment:
This is not in the scope of the pull request. I can create a seperate one if needed.